CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# How to Contribute to Eclipse Velocitas
+
+First of all, thanks for considering to contribute to Eclipse Velocitas. We really
```

Comment:
Three options
 - Call it Kuksa instead of Velocitas in this repo, the rest seems to be sufficiently generic 
 - You could make it shorter, and also copy / adapt https://github.com/eclipse/kuksa.val/blob/master/CONTRIBUTING.md (things like signed-off or maybe code guidelines are nice to mention, but int he end no one really  reads "Contributing", so probably you need to point it out in PRs on a case-by-case basis anyway
 - Leave the file out (but that is bad style, even though readership is limited :) )